src/game_constants.h

```diff
@@ -60,4 +60,9 @@
 /** Maximum move cost when handling an item */
 #define MAX_HANDLING_COST 800
 
+/** Amount by which to scale experience gain for ranged shots before any accuracy penalty is applied */
+#define RANGED_EXPERIENCE_FACTOR 20
```

Comment:
Why put these here instead of in ranged.cpp?  Things should be defined in the smallest scope possible.